rust_src/src/indent.rs

```diff
@@ -8,18 +8,19 @@ use crate::{
     buffers::{point_byte, point_min_byte},
     editfns::{insert_char, point, point_min},
     lisp::LispObject,
+    marker::buf_charpos_to_bytepos,
     multibyte::Codepoint,
+    numbers::LispNumber,
     remacs_sys::globals,
-    remacs_sys::EmacsUint,
```

Comment:
We have been separating types from functions in an attempt to reduce merge conflicts. Which means the imports should look something like:
```
remacs_sys::{EmacsInt, EmacsUint},
remacs_sys::{
    self, del_range, find_newline, position_indentation, sanitize_tab_width, scan_for_column, set_point,
    set_point_both, window_internal_height,
},
remacs_sys::Qt,
```

Reply:
BTW, window_internal_height is a good candidate for an easy PR.

Reply:
done, i also put `globals` with the other global variables (that are not symbols)